kadi_apps/app.py

```diff
@@ -53,7 +54,11 @@ def get_app(name=__name__, settings='devel'):
 
     CORS(app, supports_credentials=True)  # resources={r"*": {"origins": "http://kadi-dev:3000/*"}})
 
+    dotenv_file = Path(f'.env_{settings}')
     settings = f'kadi_apps.settings.{settings}'
+    if dotenv_file.exists():
+        logger.info(f"loading {dotenv_file.absolute()}")
+        dotenv.load_dotenv(dotenv_file)
```

Comment:
What's going on with this? Are there any implications for developers or users? Are there required documentation updates?

Reply:
The idea is that this is entirely optional. Instead of defining variables in your .bashrc, define them here, and this allows different variables for different settings.

It has no implications for anyone other than me, because I run the app with different settings sometimes.